tests/run/coverage_cmd.srctree

```diff
@@ -154,6 +158,7 @@ def run_report():
 
     assert  7 not in files['coverage_test_pyx.pyx'][-1], files['coverage_test_pyx.pyx']
     assert 12 not in files['coverage_test_pyx.pyx'][-1], files['coverage_test_pyx.pyx']
+    assert 15 not in files['coverage_test_pyx.pyx'][-1], files['coverage_test_pyx.pyx']
```

Comment:
Except that, in this case, it's because `func3()` really is not executed anywhere. ;)

I'd add a second line to it, without pragma, and make sure that gets executed.

Reply:
The pragma works in the beginning or the end of a function (also on multiple lines) but not in the middle. I must be missing something.

I mean that in
```python
def fun1():
    a = 5  # pragma: no cover
    b = 5
    return a + b

def fun2():
    a = 5
    b = 5
    return a + b  # pragma: no cover

def fun3():
    a = 5
    b = 5  # pragma: no cover
    return a + b
```
`fun1` and `fun2` produce the result I expect but `fun3` ignores the pragma.


I am working on it.